docs/project/2024/ramona.nitu/index.md

```diff
@@ -0,0 +1,103 @@
+# Morse Code Translator
+Interactive Text-to-Morse Translator with Multi-Sensory Feedback
+
+:::info 
+
+**Author**: Nițu-Săraru Ramona-Gabriela \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-GabrielaNitu
+
+:::
+
+## Description
+
+This project aims to seamlessly translate text input into Morse code, providing both visual and auditory feedback in real-time. Through the integration of an LCD screen, users can visually observe the Morse code representation of their text. Meanwhile, a buzzer emits distinct sounds for each Morse code symbol, allowing users to hear the translation simultaneously. Additionally, the RGB LED serves to provide visual feedback and indicate the status of the Morse code translation process, contributing to a more interactive and informative user experience.
+
+## Motivation
+
+I chose this project because of its relevance in various emergency aspects, such as search and rescue operations and communication in remote areas. Morse code remains a fundamental skill in these contexts, offering a reliable means of communication when other methods may be unavailable. Additionally, undertaking this project provides a valuable opportunity for personal learning and skill development. Mastering Morse code not only equips me with practical skills for emergency situations but also deepens my understanding of communication systems and encoding methods.
+
+## Architecture 
+
+1) Text Input Provided to the System: The user enters text input through a connected input device, such as a keyboard. The input text is then passed to the Morse code translator for conversion.
```

Comment:
Does this render correctly? Did you run `npm start` and looked at it?

Reply:
Yes, I used npm start and it looked ok.